futures-channel/src/oneshot.rs

```diff
@@ -331,8 +346,8 @@ impl<T> Sender<T> {
     ///
     /// # Return values
     ///
-    /// If `Ok(Ready)` is returnd then the associated `Receiver` has been dropped,
-    /// which means any work required for sending should be cancelled.
+    /// If `Ok(Ready)` is returned then the associated `Receiver` has been
+    /// dropped, which means any work required for sending should be canceled.
```

Comment:
was cancelled changed on purpose? it was correct before

Reply:
Cancelled can be spelled either way and the convention in this lib is apparently to use 'canceled'.